charts/templates/statefulset.yaml

```diff
@@ -21,6 +21,45 @@ spec:
       affinity: {{- toYaml . | nindent 8}}
       {{- end }}
       terminationGracePeriodSeconds: {{ .Values.terminationGracePeriod }}
+      {{- if ne .Values.bootstrapHostName "" }}
+      initContainers: 
+      - name: configure-group
+        image: "{{ .Values.initContainerImage.repository }}:{{ .Values.initContainerImage.tag }}"
+        command:
+          - sh
+          - '-c'
+          - |
+            echo '### Begin configuring group ###'
+
+            GROUP_CFG_TEMPLATE='{"group-name":"%s"}'
+            GROUP_CFG=$(printf "$GROUP_CFG_TEMPLATE" "$MARKLOGIC_GROUP")
+
```

Comment:
Add a logic to make sure the change group logic runs for the first node Helm chart. Like marklogic-0, but not for marklogic-1.